examples/Utils/Feeds.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Utils;
+
+use Google\Ads\GoogleAds\Lib\V2\GoogleAdsClient;
+use Google\Ads\GoogleAds\V2\Enums\FlightPlaceholderFieldEnum\FlightPlaceholderField;
+use Google\Ads\GoogleAds\V2\Resources\FeedAttribute;
+use Google\Ads\GoogleAds\V2\Resources\FeedItem;
+use Google\Ads\GoogleAds\V2\Resources\FeedItemAttributeValue;
+use Google\Ads\GoogleAds\V2\Services\GoogleAdsRow;
+
+/**
+ * Utilities that are shared between code examples related to feeds.
+ */
+final class Feeds
+{
+    const PAGE_SIZE = 1000;
+
+    /**
+     * Retrieves a feed item and its attribute values given a resource name.
+     *
+     * @param string $feedItemResourceName the feed item resource name
+     * @param int $customerId the customer ID
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @return FeedItem the feed item
+     */
+    public static function feedItemFor(
+        string $feedItemResourceName,
+        int $customerId,
+        GoogleAdsClient $googleAdsClient
+    ) {
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        // Constructs the query to get the feed item with attribute values.
+        $query = "SELECT feed_item.attribute_values FROM feed_item"
+            . " WHERE feed_item.resource_name = '$feedItemResourceName'";
+        // Issues a search request by specifying page size.
+        $response =
+            $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
+
+        // Returns the feed item attribute values, which belongs to the first item. We can ensure
+        // it belongs to the first one because we specified the feed item resource name in the
+        // query.
+        return $response->getIterator()->current()->getFeedItem();
+    }
+
+    /**
+     * Gets the index of the target feed item attribute value. This is needed to specify which feed
+     * item attribute value will be updated in the given feed item.
+     *
+     * @param FeedItemAttributeValue $targetFeedItemAttributeValue the new feed item attribute value
+     *     that will be updated
+     * @param FeedItem $feedItem the feed item that will be updated
```

Comment:
It's worth mentioning here that the caller should pass a `FeedItem` here with the `attributeValues` populated